hedera-node/hedera-mono-service/src/main/java/module-info.java

```diff
@@ -34,8 +34,6 @@
     requires grpc.stub;
     requires commons.collections4;
 
-    exports com.hedera.node.app.service.mono to
```

Comment:
let's remove the "to ...." since PR https://github.com/hashgraph/hedera-services/pull/4395 needs it for an additional module ;)

Reply:
You mean just add `exports com.hedera.node.app.service.mono` ?

Reply:
It is already defined on line 2 :)